### PR TITLE
[FIX][8.0]  pylint warning and errors

### PR DESCRIPTION
**Pull request by gurneyalex**
and hopefully make travis happy (see https://travis-ci.org/OCA/stock-logistics-warehouse/jobs/395945235)

### CODE REVIEW

File: stock_inventory_hierarchical/models/stock_inventory.py [hunk 1]
```diff
@@ -83,58 +83,60 @@ def _progress_rate(self):
...
-                       "with the state of the following parent inventories:\n"
-                       ) % ((self.name, self.state)) +
-                     "\n".join(['- %s (%s)' % (i.name, i.state)
-                                for i in inconsistent_parents])))
+        for rec in self:
+            if rec.inventory_ids:
+                inconsistent_children = self.search(
+                    [('parent_left', '>=', rec.parent_left),
+                     ('parent_right', '<=', rec.parent_right),
+                     ('state', 'not in', CONSISTANT_STATES[rec.state])])
```
Comment (elicoidal) [inline stock_inventory_hierarchical/models/stock_inventory.py]: CONSISTENT

File: stock_cycle_count/models/stock_location.py [hunk 2]
```diff
@@ -0,0 +1,109 @@
+# -*- coding: utf-8 -*-
```
Comment (elicoidal) [inline stock_cycle_count/models/stock_location.py]: `# coding: utf-8

File: stock_reserve_sale/model/sale.py [hunk 3]
```diff
@@ -27,11 +27,23 @@
 class SaleOrder(models.Model):
     _inherit = 'sale.order'
 
-    @api.multi
+    has_stock_reservation = fields.Boolean(
+        compute='_compute_stock_reservation',
+        readonly=True,
+        multi='stock_reservation',
+        store=True,
+        string='Has Stock Reservations')
```
Comment (elicoidal) [inline stock_reserve_sale/model/sale.py:35]: better finish with a comma (x2)

File: stock_reserve_sale/model/sale.py [hunk 4]
```diff
@@ -125,11 +124,20 @@ def _get_procure_method(self):
             return rule.procure_method
         return False
 
-    @api.multi
+    reservation_ids = fields.One2many(
+        'stock.reservation',
+        'sale_line_id',
+        string='Stock Reservation',
+        copy=False)
```
Comment (elicoidal) [inline stock_reserve_sale/model/sale.py:131]: same here

**Review by elicoidal — APPROVED** [review]
It is a little bit more than pylint :wink: but LGTM!

File: stock_reserve_sale/model/sale.py (hunk 3 above)
Comment (gurneyalex) [reply stock_reserve_sale/model/sale.py:35]: no need to change this just for the sake of it, IMO

**Review by simahawk — APPROVED** [review]
quick eyeball review: LG

**Pull request merged**